packages/connect/src/router.ts

```diff
@@ -131,14 +138,42 @@ export function createConnectRouter(
       );
       return this;
     },
-    rpc(service, method, implementation, options) {
-      const { protocols } = whichProtocols(options, base);
-      handlers.push(
-        createUniversalMethodHandler(
-          createMethodImplSpec(service, method, implementation),
-          protocols,
-        ),
-      );
+    rpc(
+      serviceOrMethod: ServiceType | MethodType,
+      methodOrImpl: MethodInfo | MethodImpl<MethodInfo<AnyMessage>>,
+      implementationOrOptions?:
+        | MethodImpl<MethodInfo<AnyMessage>>
+        | Partial<UniversalHandlerOptions>,
+      options?: Partial<UniversalHandlerOptions>,
```

Comment:
There's probably a better way to do, similar to how it was overridden in `packages/connect/src/implementation.ts` but I just couldn't figure it out. I tried using the same array format but it caused a weird typescript error. I assume it's because unlike in implementation, this `rpc` is not declared near the function overloads.